### ISSUE TITLE
Call it WP_Privacy_Compliance

### ISSUE THREAD

**Issue by lvl99** [GDPRWP/standard#2]
Just a minor (or major?) suggestion is to call the interface `WP_Privacy_Compliance`. Niggling semantics issue, really.

I would say that while this interface/plugin is being developed separate to WordPress, it will probably be integrated into core at some point in the future. I would suggest the `WP_` prefix and add `Compliance` to the end to indicate that this interface is to do with actions related to compliance of the GDPR.

**Comment by petersuhm** [comment]
@lvl99 good points! The naming so far is nowhere near settled upon. Thanks for chiming in!

**Comment by ePascalC** [comment]
I fully support the idea that privacy is a better choice then gdpr. Of course GDPR is now in everbodies mind (well at least in Europe), but privacy compliance should not only be towards the GDPR regulation.
So for future integrations with also e.g. the ePrivacy Directive, drop the pure gdpr pre- or suffix.

**Comment by dejliglama** [comment]
Good point! We will certantly adapt naming to a broarder privacy concern, rather than focusing on GDPR alone. However, I think we should wait with the WP_ prefix until Automattic is looking this way ;)  CC: @jespervnielsen
